kubric/custom_types.py

```diff
@@ -25,3 +26,5 @@
 PathLike = Union[str, tfds.core.ReadWritePath]
 
 ArrayLike = Union[Sequence[float], np.ndarray]
+
+Quaternion = pyquat.Quaternion
```

Comment:
```suggestion
```

custom_types is meant more as a place for type-hinting  types

Reply:
in that case, should we have another "types" module, where all the default types are specified?